README.md

```diff
@@ -46,6 +46,41 @@ usage:
 ```
 
 
+## Creating tables
+
+The `--output-table <file.tex>` flag converts input data into a LaTeX / PDF
+table. Conversion to PDF requires `pdflatex` to be installed. `bin/warmup_stats`
+also needs the names of the language and VM under test, and the output of
+`uname -a` on the machine the benchmarks were run on. Example usage:
+
+```
+./bin/warmup_stats  --output-table table.tex -l javascript -v V8 -u "`uname -a`" results.csv
```

Comment:
Can I suggest we use `bin/warmup_stats` not `./bin/warmup_stats` (and similarly below). `./` might confuse some people.